inc/integrations/class-form-utils.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace ThemeIsle\GutenbergBlocks\Integration;
+
+/**
+ * Form Utils
+ *
+ * @since 2.0.3
+ */
+class Form_Utils {
+
+	/**
+	 * Generate a random email address.
+	 *
+	 * @return string
+	 * @since 2.0.3
+	 */
+	public static function generate_test_email() {
+
+		$words = array(
+			'alfa',
+			'bravo',
+			'charlie',
+			'delta',
+			'echo',
+			'foxtrot',
+			'golf',
+			'one',
+			'two',
+			'three',
+			'four',
+			'five',
+			'six',
+			'seven',
+			'eight',
+			'nine',
+			'ten',
+		);
+
+		return 'Otter-Form-successfully-connected.delete-on-confirmation' . '.' . $words[ rand( 0, count( $words ) ) ] . '.' . $words[ rand( 0, count( $words ) ) ] . '.' . $words[ rand( 2, count( $words ) ) - 1 ] . '@otter-blocks.com';
```

Comment:
It's better to send the test email from the website email, not some random one if we don't want it to end up in spam.

Reply:
We need to use some random email every time you press the button. If you press once, the test email will be registered; if you press a second time, you will get an error that the email is already registered. Should we interpret din error as a success?